projects/plugins/boost/app/assets/src/js/features/page-cache/health/lib/get-error-data.tsx

```diff
@@ -34,12 +29,16 @@ const messages: { [ key: string ]: { title: string; message: React.ReactNode } }
 		),
 	},
 	'wp-content-not-writable': {
-		title: 'wp-content not writable',
+		title: sprintf(
+			// translators: %s refers to wp-content.
+			__( `Your site's %s folder doesn't allow updates`, 'jetpack-boost' ),
+			'wp-content'
+		),
 		message: createInterpolateElement(
 			sprintf(
 				// translators: %s refers to wp-content.
 				__(
-					`This feature cannot be enabled because <code>%s</code> is not writable. <link>Learn more.</link>`,
+					`This feature cannot be enabled because due to a filesystem permission issue Boost cannot create or modify files in <code>%s</code>. To get help solving this problem <link>click here.</link>`,
```

Comment:
```suggestion
					`This feature cannot be enabled because Jetpack Boost cannot create or modify files in <code>%s</code>. To learn more about this, please <link>click here.</link>`,
```